webapp/src/reducers.ts

```diff
@@ -565,7 +565,7 @@ const voiceChannelCalls = (state: { [channelID: string]: callState } = {}, actio
             [action.data.channelID]: {
                 ...state[action.data.channelID],
                 hostID: action.data.hostID,
-                hostChangeAt: Date.now(),
+                hostChangeAt: action.data.hostChangeAt || state[action.data.channelID].hostChangeAt,
```

Comment:
This caused some confusion. If I had a 'shaking fist' emoji. :)

Reply:
Indeed. If the timestamp was sent by the server the bug would have been avoided in the first place.

Reply:
Yep, sorry I misunderstood what you meant.  👍